pytripgui/plan_vc/plan_view.py

```diff
@@ -61,17 +61,25 @@ def uuid(self):
     def uuid(self, uuid):
         self.ui.uuid_lineEdit.setText(uuid)
 
-    def add_target_roi_with_name(self, target, target_name):
-        self.ui.targetROI_comboBox.addItem(target_name, target)
+    def add_target_roi_with_name(self, target, target_name, checked=False):
+        target_item = QRadioButton()
+        target_item.setText(target_name)
+        if checked:
+            target_item.setChecked(Qt.Checked)
 
-    def select_target_roi_to_this(self, target):
-        index_of_target = self.ui.targetROI_comboBox.findData(target, Qt.UserRole)
-        if index_of_target == -1:
-            raise Exception("Given target roi wasn't found on the list")
-        self.ui.targetROI_comboBox.setCurrentIndex(index_of_target)
+        item = QListWidgetItem(self.ui.targetROI_listWidget)
+        item.setData(Qt.UserRole, target)
+
+        self.ui.targetROI_listWidget.setItemWidget(item, target_item)
 
     def get_selected_target_roi(self):
-        return self.ui.targetROI_comboBox.currentData()
+        item_count = self.ui.targetROI_listWidget.count()
+        print(item_count)
+        for i in range(item_count):
```

Comment:
Are you force to use iteration with index `i` ? I suspect there should be some iterator available in `self.ui.targetROI_listWidget`